settings.py

```diff
@@ -108,6 +108,28 @@ def debug_toolbar_enabled(request):
                                    'test-api-key.txt'),
 }
 
+FXA_ALL_CONFIG = {
+    'cd5a21fafacc4744': {
+        'client_id': 'cd5a21fafacc4744',
+        'client_secret':
+            '4db6f78940c6653d5b0d2adced8caf6c6fd8fd4f2a3a448da927a54daba7d401',
+        'redirect_uri': 'http://olympia.dev/fxa-authorize',
+        'oauth_uri': 'https://oauth-stable.dev.lcip.org/v1',
+        'profile_uri': 'https://stable.dev.lcip.org/profile/v1',
+        'scope': 'profile',
+    },
+    'a1dc6167e57ad856': {
+        'client_id': 'a1dc6167e57ad856',
+        'client_secret':
+            '5efeeb01a523166db5d6373b7558bd74cd754376eed95837a50df5fbbb491427',
+        'redirect_uri': 'http://localhost:8000/fxa-authorize',
```

Comment:
I don't think this `localhost:8000` client should be included. Non-docker people (good luck to them) can easily add it to their local settings.